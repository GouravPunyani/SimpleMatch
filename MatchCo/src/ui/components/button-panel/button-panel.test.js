// @flow

import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'
import Comp from './button-panel'

test('renders correctly', () => {
  const tree = renderer.create(<Comp />)
  expect(tree).toMatchSnapshot()
})
