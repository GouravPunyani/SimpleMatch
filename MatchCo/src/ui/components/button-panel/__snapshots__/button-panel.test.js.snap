// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "justifyContent": "center",
      "margin": 10,
    }
  }
/>
`;
